Source/Menu/Options.Designer.cs

```diff
@@ -61,6 +61,7 @@ private void InitializeComponent()
             this.numericSoundDetailLevel = new System.Windows.Forms.NumericUpDown();
             this.checkMSTSBINSound = new System.Windows.Forms.CheckBox();
             this.tabPageVideo = new System.Windows.Forms.TabPage();
+            this.checkEnableMultisampling = new System.Windows.Forms.CheckBox();
```

Comment:
The problem is that with multisampling the MG version generally is quite slower than the XNA version.

Reply:
My preference in that case would be to configure MonoGame with a more-similar option to XNA to avoid a drastic change in performance.

A multisampling option hasn't been put on the roadmap/blueprints, nor discussed as part of the migration, so it is a surprise to me. If you don't wish to remove it, please create a forum discussion to ensure the community is okay with adding the feature (I assume they are, but this is the process we've got).

Reply:
Hi James,
yes, I agree on the public discussion. Having this on the forum allows also for the insertion of pictures and for tests by other people. I started discussion here http://www.elvastower.com/forums/index.php?/topic/33142-or-mg-and-multisampling/ and hope that there will be comments by you and other people.